### PR TITLE
LPS-133018 frontend-js-tabs-support-web Tab content and trigger no lo…

### PR DESCRIPTION
**Pull request by pat270**
…nger function due to hide listener being instantiated multiple times on click

@liferay-frontend can we get this in for next release?

https://issues.liferay.com/browse/LPS-133018

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/TabsProvider.js [hunk 1]
```diff
@@ -103,11 +103,15 @@ class TabsProvider {
...
+			const onHidden = (event) => {
 				if (event.panel === activePanel) {
 					this.show({panel, trigger});
 				}
-			});
+
+				Liferay.detach(this.EVENT_HIDDEN, onHidden);
+			};
+
+			Liferay.on(this.EVENT_HIDDEN, onHidden);
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/TabsProvider.js:114]: We can simplify this by using `Liferay.once`

**Review by markocikos — COMMENTED** [review]
See inline comment to simplify code. In principle, looks good! 

We should definitely send this in for the next release.

**Comment by pat270** [comment]
ci:test:sf

**Comment by pat270** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/TabsProvider.js (hunk 1 above)
Comment (pat270) [reply modules/apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/TabsProvider.js]: @markocikos I updated the pr

**Comment by nhpatt** [comment]
ci:forward

**Comment by diegonvs** [comment]
[This](https://github.com/liferay-frontend/liferay-portal/pull/1247#issuecomment-880015382) CI failure looks like the same as 
 https://github.com/liferay-frontend/liferay-portal/pull/1070#issuecomment-879929416. It looks unrelated. So, I'll re-run `ci:forward` again

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**